app/views/data_sources/index.haml

```diff
@@ -43,7 +47,13 @@
             .ds{class: "ds-color-#{data_source.id}"}= data_source.short_name
           %td= number_with_delimiter(@client_counts[data_source.id])
           %td= number_with_delimiter(@project_counts[data_source.id])
-          %td.text-center= checkmark(data_source.visible_in_window)
+          %td.text-center
+            - if hmis_enabled?
+              - if data_source.hmis?
+                = link_to "https://#{data_source.hmis}" do
+                  %i.icon-link-ext
+            - else
+              = checkmark(data_source.visible_in_window)
```

Comment:
Would the OP HMIS never be configured as "visible in the window?

Reply:
So, once everyone is on Access Controls, there should be no reason to mark a data source as "Visible in the Window."  I went with OP HMIS because it seems odd to have a column called HMIS where 90% of those data sources represent an HMIS.  

Happy to entertain other wording.  Perhaps it's too early to remove the "window" column.  I'll add that back for now.  I guess if we're only showing the HMIS column to installations that have at least one "OP" HMIS, then it's probably fine to just call it HMIS.